webapp-django/crashstats/topcrashers/views.py

```diff
@@ -305,11 +317,22 @@ def topcrashers(request, days=None, possible_days=None, default_context=None):
 
         crash['correlation_os'] = max(crash_counts)[1]
         sig = crash['signature']
+
+        # Augment with bugs.
         if sig in bugs:
             if 'bugs' in crash:
                 crash['bugs'].extend(bugs[sig])
             else:
                 crash['bugs'] = bugs[sig]
+
+        # Augment with first appearance dates.
+        if sig in sig_date_data:
+            first_date = isodate.parse_datetime(sig_date_data[sig])
+            crash['first_report'] = first_date.strftime('%Y-%m-%d')
+            crash['first_report_exact'] = first_date.strftime(
+                '%Y-%m-%d %H:%M:%S'
+            )
```

Comment:
Date formatting belongs in templates. I'd much prefer if you do `crash['first_report'] = first_date` and then in the template you format it nicely. 

Also, what about timezone? Do we not get any?